noxfile.py

```diff
@@ -46,7 +46,7 @@ def run_pybamm_requires(session):
     """Download, compile, and install the build-time requirements for Linux and macOS: the SuiteSparse and SUNDIALS libraries."""  # noqa: E501
     set_environment_variables(PYBAMM_ENV, session=session)
     if sys.platform != "win32":
-        session.run_always("pip", "install", "wget", "cmake")
+        session.install("wget", "cmake" , silent=False)
```

Comment:
`nox` still doesn't have a `nox.options.verbose` configuration option, however, at least un-silencing fixes the `pip` output